### ISSUE TITLE
pc.load channel not responsive

### ISSUE THREAD

**Issue by ramolli** [PizzaConsole/DiscordCopyBot#3]
Hello! This bot is an absolute gem. I can't pay money for the others, those as you mentioned to another issue, was limited. This bot is vital for us as we are changing our server massively; getting rid of old roleplay channels and chat rooms that we no longer need.

We are trying to upload all the channels we wish to get rid of into our backup server. It has been successful for one channel after inputing the **pc.copychannel** command.

However, I do not understand how to do the pc.loadchannel command and then accidentally made the bot load all the ids once more-- aha. Is there anything I need to do to make it work? Because nothing shows up after that.

Thank you so much, your bot is a godsend.

**Comment by PizzaConsole** [comment]
I am a little confused as to what the questions really is, but basically how the **pc.copychannel** command works is by reading two ids (Server ID is the id of the server where the messages currently reside and Channel ID is the channel where the messages currently reside) that you give it the serverId and the channelId: `pc.loadchannel {serverId} {channelId}` 

So you will paste that command in the NEW server and it will ask you for a channel name. Once you name the channel then it will start pasting the messages from the OLD channel in the OLD Server to the NEW Channel in the NEW Server

**Comment by ramolli** [comment]
Thank you so much for your response! I was asking how to work that command because I couldn't get it right.

pc.loadchannel {serverId} {channelId}

So this means I put the id of the server where the channel comes from, then the id of the channel I want to copy, and put that command into the new server, right?

// I GOT IT TO WORK OMG- I seriously should think more! Once again, thank you so much for this, and I realized what I had to do.
